src/platform/Zephyr/PlatformManagerImpl.cpp

```diff
@@ -88,41 +88,5 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack(void)
     return err;
 }
 
-CHIP_ERROR PlatformManagerImpl::_GetCurrentHeapFree(uint64_t & currentHeapFree)
```

Comment:
Please add `SetDiagnosticDataProvider(&DiagnosticDataProviderImpl::GetDefaultInstance());` in `InitChipStack` method above.